types/react-dev-utils/WebpackDevServerUtils.d.ts

```diff
@@ -16,25 +16,66 @@ export interface Urls {
  */
 export function choosePort(host: string, defaultPort: number): Promise<number | null>;
 
+export interface CreateCompilerOptions {
+    /**
+     * The name that will be printed to the terminal.
+     */
+    appName: string;
+    /**
+     * The webpack configuration options to be provided to the webpack constructor.
+     */
+    config: webpack.Configuration;
+    /**
+     * To provide the `urls` argument, use `prepareUrls()` described below.
+     */
+    urls: Urls;
+    /**
+     * If `true`; yarn instructions will be emitted in the terminal instead of npm.
+     */
+    useYarn?: boolean;
+    /**
+     * Takes the `require('webpack')` entry point.
+     */
+    webpack: typeof webpack;
+}
+export type CreateCompilerOptionsTypescript = CreateCompilerOptions & {
```

Comment:
Can't it also be an interface (extending `CreateCompilerOptions`) for consistency with `CreateCompilerOptions`?